docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,8 +1,55 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-# Local agents
\ No newline at end of file
+# Running a Dagster+ agent locally
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) locally. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent (like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
+
+## Installation
+
+
+### Step 1: Install dagster-cloud
+
+```shell
+pip install dagster-cloud
+```
+
+## Step 2: Configure the agent
+
+[Generate an agent token](dagster-plus/deployment/tokens) and configure your agent
+
+```shell
+dagster-cloud configure
+```
+
+## Step 3: Run the agent
+
+```
+dagster-cloud agent run
```

Comment:
Following these steps exactly and I got:

> No directory provided or DAGSTER_CLOUD environment variable set. You must supply the location of a directory containing a dagster.yaml file as a parameter or by setting the DAGSTER_HOME environment variable.

should the step on `dagster.yaml` be moved to step 3 and step 4 be to run the agent?

Reply:
when I added the dagster_home argument (`dagster-cloud agent run .` in my case) I got 

I got errors until I put this in the `dagster.yaml` (taken from the old docs):

```yaml
dagster_cloud_api:
  agent_token: <YOUR_AGENT_TOKEN>
  deployment: prod

user_code_launcher:
  module: dagster_cloud.workspace.user_code_launcher
  class: ProcessUserCodeLauncher
```